pkg/controller/cyclenoderequest/transitioner/transitions_pending_test.go

```diff
@@ -293,10 +294,65 @@ func TestPendingNoKubeNodes(t *testing.T) {
 	assert.Equal(t, v1.CycleNodeRequestHealing, cnr.Status.Phase)
 }
 
-// Test to ensure that Cyclops will not proceed if there is node detached from
-// the nodegroup on the cloud provider. It should try to wait for the issue to
-// resolve to transition to the Healing phase if it doesn't.
-func TestPendingDetachedCloudProviderNode(t *testing.T) {
+// Test to ensure that node objects that exist in the cluster without a matching
+// instance in the cloud provider are cleaned up and then cycling can proceed
+// as normal.
+func TestPendingOrphanedKubeNodes(t *testing.T) {
```

Comment:
Is there a reason the tests can't all go in one `_test.go` file and just be named and commented appropriately?

Reply:
I can see a lot of tests being written to different phases, i'd like to not have one mega file with all the tests that becomes hard to navigate.